aviary/variable_info/enums.py

```diff
@@ -59,6 +59,7 @@ class EquationsOfMotion(Enum):
     """
     HEIGHT_ENERGY = 'height_energy'
     TWO_DEGREES_OF_FREEDOM = '2DOF'
+    SOLVED_2DOF = 'solved_2DOF'
     SOLVED = 'solved'
```

Comment:
Does this mean "SOLVED" is actually solved height energy?

Reply:
I think "SOLVED" is GASP based 2DOF and "SOLVED_2DOF" is FLOPS based